lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -422,6 +422,17 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Path to <b>newrelic.yml</b>. If undefined, the agent checks the following directories (in order): <b>config/newrelic.yml</b>, <b>newrelic.yml</b>, <b>$HOME/.newrelic/newrelic.yml</b> and <b>$HOME/newrelic.yml</b>.'
         },
+        :'custom_events_instrumentation_topics' => {
+          :default => [],
+          :public => true,
+          :type => Array,
+          :allowed_from_server => true,
```

Comment:
This should probably be set to false.
I don't think the server would be sending anything for it anyways since that isn't something the server knows about. 
Plus, from what I understand, this config is only used during startup when we are installing instrumentation. Configs that are only used right at startup can't be allowed_from_server because the agent would have already finished setup by the time the server response is received.

Reply:
Agreed! Good point!

Reply:
Updated with https://github.com/newrelic/newrelic-ruby-agent/pull/1659/commits/951476b03bd2beafa3f9804a04b79b0b8a88ba6f